### PR TITLE
filefrag: add page

### PR DESCRIPTION
**Pull request by teeteejo**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/filefrag.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# filefrag
+
+> Report how badly fragmented a particular file might be.
+> More information: <https://linux.die.net/man/8/filefrag>.
+
+- Report on file fragmentation:
+
+`filefrag {{file}}`
```
Comment (navarroaxel) [inline pages/linux/filefrag.md:8]:
```suggestion
`filefrag {{path/to/file}}`
```

File: pages/linux/filefrag.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# filefrag
+
+> Report how badly fragmented a particular file might be.
+> More information: <https://linux.die.net/man/8/filefrag>.
+
+- Report on file fragmentation:
```
Comment (navarroaxel) [inline pages/linux/filefrag.md:6]:
```suggestion
- Display a report for a specific file:
```

File: pages/linux/filefrag.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+`filefrag {{file}}`
+
+- Generate report for multiple files:
```
Comment (navarroaxel) [inline pages/linux/filefrag.md:10]:
```suggestion
- Display a report for space-separated list of files:
```

File: pages/linux/filefrag.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+`filefrag {{file1}} {{file2}}`
```
Comment (navarroaxel) [inline pages/linux/filefrag.md:12]:
```suggestion
`filefrag {{path/to/file1}} {{path/to/file2}}`
```

File: pages/linux/filefrag.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+- Use 1024 byte blocksize for the output:
```
Comment (navarroaxel) [inline pages/linux/filefrag.md:14]:
```suggestion
- Display a report using a 1024 byte blocksize:
```

File: pages/linux/filefrag.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+`filefrag -b {{files}}`
```
Comment (navarroaxel) [inline pages/linux/filefrag.md:16]:
```suggestion
`filefrag -b {{path/to/file}}`
```

File: pages/linux/filefrag.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+- Sync the file before requesting the mapping:
+
+`filefrag -s {{files}}`
```
Comment (navarroaxel) [inline pages/linux/filefrag.md:20]:
```suggestion
`filefrag -s {{path/to/files}}`
```

File: pages/linux/filefrag.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+- Display mapping of extended attributes:
+
+`filefrag -x {{files}}`
```
Comment (navarroaxel) [inline pages/linux/filefrag.md:24]:
```suggestion
`filefrag -x {{path/to/files}}`
```

File: pages/linux/filefrag.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+
+- Be verbose when checking for file fragmentation:
+
+`filefrag -v {{files}}`
```
Comment (navarroaxel) [inline pages/linux/filefrag.md:28]:
```suggestion
`filefrag -v {{path/to/files}}`
```

File: pages/linux/filefrag.md [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+
+- Sync the file before requesting the mapping:
+
+`filefrag -s {{files}}`
+
+- Display mapping of extended attributes:
+
+`filefrag -x {{files}}`
+
+- Be verbose when checking for file fragmentation:
```
Comment (navarroaxel) [inline pages/linux/filefrag.md:26]:
```suggestion
- Display a report with verbose information:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hey @teeteejo, you look working hard here. I left some comments.

**Comment by teeteejo** [comment]
Thanks, this is fun :)

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Thanks, @teeteejo :tada:

**Comment by teeteejo** [comment]
Thank you. This relates to #4250

**Pull request merged**